telethon_generator/data/errors.csv

```diff
@@ -175,6 +175,7 @@ MAX_QTS_INVALID,400,The provided QTS were invalid
 MD5_CHECKSUM_INVALID,400,The MD5 check-sums do not match
 MEDIA_CAPTION_TOO_LONG,400,The caption is too long
 MEDIA_EMPTY,400,The provided media object is invalid or the current account may not be able to send it (such as games as users)
+MEDIA_GROUPED_INVALID,400,The Grouped Media is Invalid
```

Comment:
I'd rather not provide any description if it's just going to repeat the error message (I know there are other older cases doing this, but I want to move away from that):

```suggestion
MEDIA_GROUPED_INVALID,400,
```